cmd/web5/cmd_did.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/tbd54566975/web5-go/dids"
+)
+
+type didResolveCmd struct {
+	URI string `arg:"" name:"uri" help:"The URI to resolve."`
+}
+
+func (c *didResolveCmd) Run(_ context.Context) error {
+	result, err := dids.Resolve(c.URI)
+	if err != nil {
+		return err
+	}
+
+	jsonDID, err := json.MarshalIndent(result.Document, "", "  ")
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(string(jsonDID))
```

Comment:
makes me wonder if  this needs some sort of message or prettifying

Reply:
Yeah right now we're `json.MarshalIndent()`ing to make it pretty, but we could push this down to the SDK-level 

Tangentially, I have a ticket for adding CLI args so the user can direct formatting https://github.com/TBD54566975/web5-go/issues/64

Reply:
How were you thinking about exposing this at the sdk level @KendallWeihe ?

Reply:
Good point, reiterating that the CLI is merely a wrapper around SDK functionality (it doesn't extend it)

https://github.com/TBD54566975/web5-go/issues/86